src/lib/core.c

```diff
@@ -894,6 +894,60 @@ int gpiod_line_event_read_fd(int fd, struct gpiod_line_event *event)
 	return 0;
 }
 
+static struct gpiod_chip *
```

Comment:
If you're using the gpiod_ prefix, export the function. Any reason why we wouldn't want it exported?